<launch>
  <!--<node name="usb_cam" pkg="ros_astra_launch" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>-->
  <node name="ros_face_recognition" pkg="ros_face_recognition" type="face_recognition_node.py" respawn="true" output="screen">
    <param name="show_window" value="True" />
    <param name="tracker_quality" value="5" />
    <param name="scale" value="0.5" />
  </node>
</launch>
